tripgo.swagger.yaml

```diff
@@ -90,6 +90,110 @@ paths:
           schema:
             $ref: '#/definitions/RegionInfoResponse'
 
+  /regionDetails.json:
+    post:
+      tags:
+        - Configuration
+      summary: Detailed TSPs per region
+      description:
+        Retrieves detailed information about covered transport service providers for a specified region.
+      parameters:
+        - name: input
+          in: body
+          schema:
+            type: object
+            properties:
+              region:
+                type: string
+                description: Region name/code from `regions.json`
+              modes:
+                type: array
+                items:
+                  type: string
+                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](http://skedgo.github.io/tripgo-api/site/advanced/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included.
+              operators:
+                type: array
+                items:
+                  type: string
+                description: TSPs name for which results should be returned. In absence, all operators are included.
+              routes:
+                type: array
+                items:
+                  type: string
+                description: Routes short name for which results should be returned. In absence, all routes are included.
+            required:
+              - region
+            example:
+              - region: AU_NSW_Sydney
+                modes:
+                  - pt_pub_tram
+                  - pt_pub_bus
+                operators:
+                  - Metro - Los Angeles
+                routes:
+                  - Metro Blue Line
+                  - 205
+                  - 620
+                  - Metro Purple Line
+      responses:
+        200:
+          description: Successful response
+          schema:
+            type: object
+            properties:
+              code:
+                type: string
+                description: Internal identifier for the region.
+              transitModes:
+                type: array
+                description: Detailed TSPs information per public transit modes.
+                items:
+                  type: object
+                  properties:
+                    identifier:
+                      type: string
+                      description: A specific public transit mode-identifier, e.g., `pt_pub_bus` or `pt_pub_gondola`.
+                    numberOfServices:
+                      type: intiger
+                      description: Total number of services for this region's public transport mode.
+                    realTime:
+                      type: string
+                      description: SHOULD I PUT SOME GLOBAL MODE STATS HERE?
```

Comment:
ok, still if we have to go down to routes in stats this would no represent much effort